### PR TITLE
Scripts for creating releases using Github actions.

### PR DESCRIPTION
**Pull request by hsorby**
Using actions to help with creating releases.

### CODE REVIEW

**Review by nickerso — APPROVED** [review]

File: .github/workflows/deploy-on-release.yml [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+
+env:
+  BUILD_TYPE: Release
+  MACOSX_DEPLOYMENT_TARGET: 10.14
+
+jobs:
+  many-linux-wheels:
+    name: ${{ matrix.config.name }}
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64
```
Comment (agarny) [inline .github/workflows/deploy-on-release.yml:16]: I guess you are aware of the message at https://quay.io/pypa/manylinux2014_x86_64, i.e.
> On June 30 2021, Quay.io will move to Red Hat Single Sign-On Services. If you haven't done so already, please [create a PERSONAL (not corporate) Red Hat account](https://www.redhat.com/wapps/ugc/register.html) and attach it to your Quay.io account, following [these instructions](https://access.redhat.com/articles/5363231). If you create a corporate account you WILL lose access to your Quay personal namespace

File: .github/workflows/deploy-on-release.yml [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+        mkdir $src_dir
+
+        # fetch libraries
+        git clone https://github.com/hsorby/l1bcellml -b master --depth 1 $src_dir/libcellml
+        git clone https://github.com/OpenCMISS-Dependencies/zlib -b v1.2.3 --depth 1 $src_dir/zlib
+        git clone https://github.com/OpenCMISS-Dependencies/libxml2 -b v2.9.6 --depth 1 $src_dir/libxml2
+
+        curl -L https://cfhcable.dl.sourceforge.net/project/swig/swig/swig-4.0.2/swig-4.0.2.tar.gz > $downloads_dir/swig-4.0.2.tar.gz
+        curl -L ${{ matrix.config.python_tarball }} > $downloads_dir/Python.tar.gz
+        curl -L https://www.openssl.org/source/openssl-1.1.1h.tar.gz > $downloads_dir/openssl-1.1.1h.tar.gz
```
Comment (agarny) [inline .github/workflows/deploy-on-release.yml:50]: Just in case, OpenSSL 1.1.1j is available and 1.1.1k is about to be released.

Comment (agarny) [reply .github/workflows/deploy-on-release.yml:50]: Is it worth checking the version of our other dependencies?

**Review by agarny — APPROVED** [review]
A couple of minor comments, but otherwise will assume it all works fine. IOW, would be nice to have confirmation of what is generated.

**Pull request merged**